vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -0,0 +1,75 @@
+# Entando Glossary
+
+## Entando Terms
+
+### Core Products
```

Comment:
Let's simplify this a bit with an Entando Section, alpha-sorted (it's close now), and then General Technical Terms.